src/bazel/bazel_quickpick.ts

```diff
@@ -146,8 +166,8 @@ export async function queryQuickPickPackage({
 
   const packagePaths = await new BazelQuery(
     getDefaultBazelExecutablePath(),
-    workspaceInfo.workspaceFolder.uri.fsPath,
-  ).queryPackages(query ?? "//...");
+    getWorkingDirectory(workspaceInfo),
```

Comment:
Is it actually expected to limit the results to just the current package for this `queryQuickPickPackage` function? This means that the result will always be of length 1